amundsen_application/base/examples/example_superset_preview_client.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import requests
+import uuid
+
+from requests import Response
+from typing import Any, Dict  # noqa: F401
+
+from amundsen_application.base.base_superset_preview_client import BaseSupersetPreviewClient
+
+DEFAULT_DATABASE_MAP = {
```

Comment:
The repo has a directory for example/docker/neo4j/conf(maybe we should move it to metadata), but it would be good to put this file under the same example dir.

Reply:
My understanding was that we would move this file to the parent repository? I do like this file here though, as it makes sense to me to have examples for our abstract base clients in the base directory.

Reply:
sg